sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -7,6 +7,8 @@
     <PackageTags>Azure;Event Hubs;EventHubs;.NET;AMQP;IoT;$(PackageCommonTags)</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsTrimmable Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))">true</IsTrimmable>
+    <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">true</IsAotCompatible>
```

Comment:
What do these do without a net6.0/net7.0 target?

Reply:
I'm on the fence on whether we should make this change or not. It feels like either we should build for a compatible TFM or drop these properties.

The concern is that we check this in now, some one adds warnings to the assembly but that doesn't get caught since these analyzers aren't running. Then when the next person tries to add a `net8.0` TFM they are blocked because these analyzers start emitting warnings.

Reply:
@eerhardt: For context, we're in the process of adding a `net8` target across the Azure SDK packages, so this will be active in the short term.

Reply:
That's good information. Given that, then I would lean towards leaving these lines in.